HIRS_Utils/src/test/java/hirs/ima/matching/ImaAcceptableHashRecordMatcherTest.java

```diff
@@ -0,0 +1,153 @@
+package hirs.ima.matching;
+
+import hirs.data.persist.Digest;
+import hirs.data.persist.IMABaselineRecord;
+import hirs.data.persist.IMAMeasurementRecord;
+import hirs.data.persist.ReportMatchStatus;
+import hirs.data.persist.SimpleImaBaseline;
+import hirs.data.persist.SimpleImaBaselineTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Tests for the ImaAcceptableHashRecordMatcher.  These are basic tests of its functionality;
+ * more complete tests for contains() as used operationally by baselines that test various
+ * permutations of parameters are located in SimpleImaBaselineTest, BroadRepoImaBaselineTest,
+ * TargetedRepoImaBaselineTest, ImaBlacklistBaselineTest, and ImaIgnoreSetBaselineTest.
+ */
+public class ImaAcceptableHashRecordMatcherTest {
+    private static final String FILENAME_1 = "/bin/ls";
+    private static final String FILENAME_2 = "/bin/ls_with_another_name";
+    private static final Digest HASH_1 =
+            SimpleImaBaselineTest.getDigest("33333c2f7f3003d2e4baddc46ed4763a49543333");
+    private static final Digest HASH_2 =
+            SimpleImaBaselineTest.getDigest("00000c2f7f3003d2e4baddc46ed4763a49543333");
+
+    /**
+     * Tests that contains functions if no records are given.
```

Comment:
All of the test function javadocs begin with "Tests that contains", which is a bit confusing English until you realize that the word "contains" in the comment refers to a method. Maybe put some parenthesis after the word contains to be clearer.

Reply:
Will do.

Reply:
ok